README.md

```diff
@@ -14,54 +13,39 @@ If you want to use it directly in a browser use a CDN like [Skypack](https://www
 ```javascript
 import urlJoin from 'url-join';
 
-const id = '42'
-const postUrl = urlJoin('/posts', id, '/metadata/');
-// => "/posts/42/metadata"
+// Create a Full URL
+const fullUrl = urlJoin('http://www.google.com', 'a', '/b/cd', '?foo=123', '&bar=456', '#heading-1');
+fullUrl.toString(); // 'http://www.google.com/a/b/cd?foo=123&bar=456#heading-1'
```

Comment:
There should be no need to call `.toString()` here, as the returned value is already a string.